Sources/TuistDependencies/SwiftPackageManager/Utils/PackageInfoMapper.swift

```diff
@@ -1437,10 +1437,10 @@ extension PackageInfoMapper {
     }
 }
 
-extension ProjectDescription.TargetDependency.Condition {
+extension ProjectDescription.PlatformCondition {
     struct OnlyConditionsWithUnsupportedPlatforms: Error {}
 
-    /// Map from a package condition to ProjectDescription.TargetDependency.Condition
+    /// Map from a package condition to ProjectDescription.InclusionCondition
```

Comment:
This comment should be updated.
```suggestion
    /// Map from a package condition to ProjectDescription.PlatformCondition
```